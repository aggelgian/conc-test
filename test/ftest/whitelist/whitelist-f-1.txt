{whitelist,just_loop,1}.
