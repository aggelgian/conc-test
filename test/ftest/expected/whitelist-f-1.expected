Testing whitelist:f/1 ...
=== Inputs That Lead to Runtime Errors ===
#1 whitelist:f(424242)
